uk.co.compucorp.civicrm.tasksassignments/CRM/Tasksassignments/Upgrader.php

```diff
@@ -468,21 +468,48 @@ public function upgrade_1017()
 
     return false;
   }
-    
+
+  /**
+   * Make sure that the Tasks and Assignments main menu item
+   * (and subsequently the Dashboard menu item) are restricted to only the users
+   * with 'access Tasks and Assignments' permission
+   */
+  public function upgrade_1018() {
```

Comment:
we could update the current civihr setups manually ... so I think there is no need for this new upgrader .. what do you think ?